emottak-utils/src/main/kotlin/no/nav/emottak/utils/events/model/EventType.kt

```diff
@@ -0,0 +1,113 @@
+package no.nav.emottak.utils.events.model
+
+enum class EventType(val value: Int, val description: String) {
```

Comment:
:eyes: 

Hvordan ble event-kodene bestemt? Er de satt i en spesifikk rekkefølge, eller finnes det regler for dem som ikke er dokumentert?

Reply:
De er kopiert fra gamle emottak.